GitUI/BranchTreePanel/RepoObjectsTree.cs

```diff
@@ -275,11 +276,20 @@ public void SelectionChanged(IReadOnlyList<GitRevision> selectedRevisions)
             ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
             {
                 cancellationToken.ThrowIfCancellationRequested();
-                HashSet<string> mergedBranches = selectedGuid is null
-                    ? new HashSet<string>()
-                    : (await Module.GetMergedBranchesAsync(includeRemote: true, fullRefname: true, commit: selectedGuid)).ToHashSet();
+                HashSet<string> mergedBranches = new();
+                if (selectedGuid is not null)
+                {
+                    try
+                    {
+                        mergedBranches = (await Module.GetMergedBranchesAsync(includeRemote: true, fullRefname: true, commit: selectedGuid)).ToHashSet();
+                    }
+                    catch (ExternalOperationException)
+                    {
+                        // ignore not parsable "HEAD"
+                    }
```

Comment:
```suggestion
                HashSet<string> mergedBranches;
                if (selectedGuid is not null)
                {
                    try
                    {
                        mergedBranches = (await Module.GetMergedBranchesAsync(includeRemote: true, fullRefname: true, commit: selectedGuid)).ToHashSet();
                    }
                    catch (ExternalOperationException)
                    {
                        // ignore not parsable "HEAD"
                        mergedBranches = new();
                    }
```

Reply:
`mergedBranches` shall not be nullable, and must be an empty `HashSet` if `selectedGuid is null`.